plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/common/SubversionOutputConsolePresenter.java

```diff
@@ -58,8 +59,7 @@ public void go(final AcceptsOneWidget container) {
     }
 
     public void print(@NotNull final String text) {
-        final String[] lines = text.split("\n");
-        for (String line : lines) {
+        for (String line : Splitter.on("\n").split(text)) {
```

Comment:
I don't think we should use that as 
- it is not obvious and others can replace it back with default splitting later
- It will be compiled into JavaScript so these benchmarks are useless in this case.